README.md

```diff
@@ -43,6 +43,16 @@ These environment variables are specified using the [balenaCloud dashboard](http
 |`operatingHours` | `8-22` (hours during which the data will refresh at the interval above)
 |`screenBlankHours` | `1-6` (hours during which the screen will be blank and data will not refresh)
 | `outOfHoursName` | `London Paddington` (name shown when current time is outside the `operatingHours`)
+| `dualScreen` | `True` (if you are using two displays)
+| `screenPlatform1` | `1` (sets the platform you want to have displayed on the first display)
+| `screenPlatform2` | `2` (sets the platform you want to have displayed on the second display)
+
+
+If using multi screen the following line needs to be added into /boot/config.txt
```

Comment:
This can also be specified from the device configuration screen in the balenaCloud dashboard - pretty handy for stuff like this!

Reply:
I didn't know that, I've added it into the readme now